README.md

```diff
@@ -13,3 +13,5 @@ New York accident rate
 <br/>
 
 <b>Important: </b><i>To add local git hooks you have to execute sbt writeHooks</i>
```

Comment:
Could you please wrap sbt writeHooks statement into back ticks and bring it out from <i> to make it looks like 
`sbt writeHooks`

Reply:
done